client/components/domains/use-my-domain/utilities/get-option-info.js

```diff
@@ -138,11 +135,10 @@ export function getOptionInfo( {
 			};
 			break;
 		default:
-			transferContent = optionInfo.transferNotSupported;
-	}
-
-	if ( ! isDomainTransferrable ) {
-		transferContent = domainTransferContent;
+			transferContent = {
```

Comment:
One small suggestion here is to define `availabilityNotice` closer to where it's used so the code is easier to understand 🙂 
```suggestion
			const availabilityNotice = getAvailabilityNotice( domain, availability );
			transferContent = {
```